### DISCUSSION TITLE
Nack message with delay

### DISCUSSION THREAD

**Discussion by ludovit-ubrezi** [cloudamqp/amqp-client.js#108] [Q&A] [answered]
Hello, its possible to nack message with some delay like in rascal?
Switched to this library due to more control over consumer/publish due to Promise based code. But i saw only basic nack.

**Comment by carlhoerberg** [comment, marked as answer]
Why not use setTimeout?

**Reply by ludovit-ubrezi** [reply]
Yeah its possible to do it also when I self handle it. Its just an option to improve the library. As when you put classic nack() it shoots "hundreds" of requests to MQ. But yes i can use my own functionality for delay if i want to nack it.
